### DISCUSSION TITLE
Export to CSV rather than TSV for broader compatibility in `statspage`

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5710] [Feature Requests]
It can still use tab separators

**Comment by maxpatiiuk** [comment]
Should use the csv generation library that we are already using in the workbench as it takes care of auto escaping and can be enabled to take care of adding Bite Order Marker (needed for Excel to work with utf-8)

@realVinayak please review all the places in the code where we have a call to `downloadFile()` to identify the places that should be refactored
